downstream/modules/aap-hardening/ref-sensitive-variables-install-inventory.adoc

```diff
@@ -7,7 +7,9 @@
 
 [role="_abstract"]
 
-The installation inventory file contains a number of sensitive variables, mainly those used to set the initial passwords used by {PlatformNameShort}, that are normally kept in plain text in the inventory file. To prevent unauthorized viewing of these variables, you can keep these variables in an encrypted link:https://docs.ansible.com/ansible/latest/vault_guide/index.html[Ansible vault]. To do this, go to the installer directory and create a vault file:
+The installation inventory file contains a number of sensitive variables, mainly those used to set the initial passwords used by {PlatformNameShort}, that are normally kept in plain text in the inventory file. To prevent unauthorized viewing of these variables, you can keep these variables in an encrypted link:https://docs.ansible.com/ansible/latest/vault_guide/index.html[Ansible vault]. 
+
+To do this, go to the installer directory and create a vault file:
 
```

Comment:
.Procedure

Reply:
. Navigate to the installer directory.
. Create a vault file:

Reply:
As above